packages/connect-extension-protocol/src/index.ts

```diff
@@ -68,7 +68,7 @@ export interface ProviderMessageData {
   /** The name of the blockchain network the app is talking to **/
   chainName: string;
   /** What action the `ExtensionMessageRouter` should take **/
-  action: 'forward' | 'connect' | 'disconnect';
+  action: 'forward' | 'connect' | 'disconnect' | 'spec';
```

Comment:
I don't think we want a new `action` at this layer.  We want a message from the `ExtensionProvider` to ask the `ExtensionMessage` to **`forward`** a `'spec'` message.

Reply:
The idea behind this was to have complete different flows for spec and other json-rpc forwards for future usage - but if it makes sense to have flow until split this on the message level - I will proceed with that